src/Courses/CourseDetailsTabs/EnrollmentsTab.tsx

```diff
@@ -1,21 +1,25 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserObject } from '../CourseInterfaces';
 import EmailComponentWrapper from './EmailComponentWrapper';
+import AxiosRequest from '../../Hooks/AxiosRequest';
 
 interface EnrollmentsTabProps {
-
+    courseId: number
 }
 
-const mock_users = [
-    new UserObject({first_name: 'Scott', last_name: 'Summers', id: 1}),
-    new UserObject({first_name: 'Henry', last_name: 'McCoy', id: 2}),
-    new UserObject({first_name: 'Jean', last_name: 'Grey', id: 3}),
-    new UserObject({first_name: 'Anne', last_name: 'LeBeau', id: 4}),
-];
+export const EnrollmentsTab: React.FC<EnrollmentsTabProps> = ({courseId}) => {
+    const [users, setUsers] = useState([]);
+
+    useEffect(() => {
+        (async () => {
+            const usersResp = await AxiosRequest.get(`users?courseId=${courseId}`);
```

Comment:
We could just pass courseId in the params of the options, doesn't really matter in this case because url encoding will never be an issue and there is only one parameter but for consistent in case that isn't the case in other request I'd prefer params